ceph/ceph.py

```diff
@@ -414,11 +414,15 @@ def setup_insecure_registry(self):
         Update all ceph demons nodes to allow insecure registry use
         """
         if self.containerized and self.ansible_config.get('ceph_docker_registry'):
-            insecure_registry = '{{"insecure-registries" : ["{registry}"]}}'.format(
-                registry=self.ansible_config.get('ceph_docker_registry'))
-            logger.warn('Adding insecure registry:\n{registry}'.format(registry=insecure_registry))
-            for node in self.get_nodes():
-                node.write_docker_daemon_json(insecure_registry)
+            logger.warn('Adding insecure registry:\n{registry}'
+                        .format(registry=self.ansible_config.get('ceph_docker_registry')))
+            # for node in self.get_nodes():
```

Comment:
In the follow on PR, please remove the unneeded comments.

Reply:
sure, for now the registry we use dont need to added.